cleantalkantispam.php

```diff
@@ -256,6 +256,26 @@ private function checkIsPaid($ct_api_key = '', $force_check = false)
 		return isset($save_params) ? $save_params : null;
 	}
 
+    /**
+     * Checking curl/allow_url_fopen availability
+     */
+    private function checkCurlAUFopenAvailability() {
+        if(!function_exists('curl_init')) {
```

Comment:
We don't need to check both CURL and AUFO, we at least one.
Should be something like this:
`If( CURL )
 retrurn true|false;
if( allow_url_fopen )
 return true|false;
else
return error;`